### ISSUE TITLE
Bikeshed failures

### ISSUE THREAD

**Issue by rtoy** [WebAudio/web-audio-api#2349]
**Describe the issue**

There are new bikeshed failures noted by Travis CI:
```
LINK ERROR: No 'idl' refs found for 'MediaStream'.
{{MediaStream}}
LINK ERROR: No 'idl' refs found for 'MediaStreamTrack'.
{{MediaStreamTrack}}
LINK ERROR: No 'idl-name' refs found for 'MediaStream'.
<a data-link-type="idl-name" data-lt="MediaStream">MediaStream</a>
LINK ERROR: No 'idl-name' refs found for 'MediaStreamTrack'.
<a data-link-type="idl-name" data-lt="MediaStreamTrack">MediaStreamTrack</a>
LINK ERROR: No 'idl' refs found for 'MediaDevices'.
{{MediaDevices}}
```

**Where Is It**

See the logs at https://travis-ci.com/github/WebAudio/web-audio-api/builds/227784690Provide

**Additional Information**

This wasn't happening a couple of weeks ago.

**Comment by rtoy** [comment]
Talked to @tabatkins about this.  It an issue in bikeshed handling some changes in ReSpec docs.

**Comment by tabatkins** [comment]
Yeah this is an accidental collision of the way Bikeshed's spec-spider harvests definitions and a recent change to how ReSpec formats its definitions. Looking into it, no action required (or really possible) on your part, it'll fix itself automatically when we resolve this.

**Comment by rtoy** [comment]
I think the bikeshed issues are fixed now.  I don't see the warnings anymore after updating the bikeshed to the latest released version when I build the spec locally.

**Comment by rtoy** [comment]
Yep.  Definitely fixed, as shown by the successful merges of #2348 and #2354.

Thanks @tabatkins
